Cargo.toml

```diff
@@ -10,6 +10,6 @@ documentation = "https://docs.rs/plane-split"
 
 [dependencies]
 binary-space-partition = "0.1.2"
-euclid = "0.13"
+euclid = "~0.14.1"
```

Comment:
0.14.1 is the same thing as ^0.14.1

http://doc.crates.io/specifying-dependencies.html#specifying-dependencies-from-cratesio